workshops/geometric_pattern/README.md

```diff
@@ -561,17 +596,15 @@ This code says that if the user presses a key with the `keyCode` 115 or 83 (whic
 
 The `return false;` at the end of the function tells the browser to ignore any default behavior that might be associated with the pressed key (ex. if you press enter, the browser thinks that you're trying to submit a form by default -- this prevents that).
 
-### Making It Live
-
-Make sure all of your files are saved. Then, head on over to the terminal in Cloud9 by pressing `alt+t` and type the following commands (pressing Enter after each one):
+Now let's commit these changes:
 
 - `git add --all`
-- `git commit -m "Create a geometric pattern using p5.js"`
+- `git commit -m "Add save keybinding to geometric_pattern"`
 - `git push`
 
-Then, enter your GitHub username and password (careful here, as password will not be displayed).
+### Making It Live
 
-Congratulations! Your pattern is now live on `USERNAME.github.io/geometric_pattern/` (replace `USERNAME` with your own GitHub username!)
+Congratulations! Ever since your first commit, your pattern has been online at `USERNAME.github.io/geometric_pattern/` (replace `USERNAME` with your own GitHub username!).
```

Comment:
I'm not sure if it would be too over the top, but how about reminding the reader that they can share what they've built with their friend/family?

I feel like that will really hammer home what "live" and "online" are.